### PR TITLE
Add comments to ramp closed appeal table

### PR DESCRIPTION
**Pull request by leikkisa**
connects #9559

### CODE REVIEW

File: db/migrate/20190227002146_add_comments_to_ramp_closed_appeal.rb [hunk 1]
```diff
@@ -0,0 +1,10 @@
+class AddCommentsToRampClosedAppeal < ActiveRecord::Migration[5.1]
+  def change
+    change_table_comment(:ramp_closed_appeals, "Keeps track of legacy appeals that are closed or partially closed in VACOLS due to being transitioned to a RAMP election.  This data can be used to rollback the RAMP Election if needed.")
+    change_column_comment(:ramp_closed_appeals, :closed_on, "The date that the legacy appeal was closed in VACOLS and opted into RAMP.")
```
Comment (shanear) [inline db/migrate/20190227002146_add_comments_to_ramp_closed_appeal.rb]: `date` should be `time` or `datetime`

File: db/migrate/20190227002146_add_comments_to_ramp_closed_appeal.rb [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
+    change_column_comment(:ramp_closed_appeals, :nod_date, "The date when the Veteran filed a notice of disagreement for the original claims decision in the legacy system - the step before a Veteran receives a Statement of the Case and before they file a Form 9.")
```
Comment (shanear) [inline db/migrate/20190227002146_add_comments_to_ramp_closed_appeal.rb]: I don't think the part after the hyphen is needed. NOD should pretty ubiquitous in this context.

Comment (shanear) [inline db/migrate/20190227002146_add_comments_to_ramp_closed_appeal.rb]: `date` should be `time` or `datetime`

File: db/migrate/20190227002146_add_comments_to_ramp_closed_appeal.rb [hunk 3]
```diff
@@ -0,0 +1,10 @@
...
+    change_column_comment(:ramp_closed_appeals, :partial_closure_issue_sequence_ids, "If the entire legacy appeal could not be closed and moved to the RAMP Election, the VACOLS sequence IDs of issues on the legacy appeal which were closed are stored here, indicating that it was a partial closure.")
+    change_column_comment(:ramp_closed_appeals, :ramp_election_id, "The ID of the RAMP election that closed the legacy appeal.")
+    change_column_comment(:ramp_closed_appeals, :vacols_id, "The VACOLS ID of the legacy appeal that has been closed and opted into RAMP.")
```
Comment (shanear) [inline db/migrate/20190227002146_add_comments_to_ramp_closed_appeal.rb]: may want to specify that this correlates to `BFKEY` on the VACOLS record

File: db/migrate/20190227002146_add_comments_to_ramp_closed_appeal.rb (hunk 2 above)
Comment (pkarman) [reply db/migrate/20190227002146_add_comments_to_ramp_closed_appeal.rb]: maybe Title Case "Notice of Disagreement" so it's more clear what the column name refers to?

File: db/migrate/20190227002146_add_comments_to_ramp_closed_appeal.rb (hunk 1 above)
Comment (leikkisa) [reply db/migrate/20190227002146_add_comments_to_ramp_closed_appeal.rb]: 👍

**Pull request merged**